### ISSUE TITLE
Error building for MACHINE zc706-zynq7

### ISSUE THREAD

**Issue by gutelfuldead** [NobuoTsukamoto/meta-tensorflow-lite#56]
Building for MACHINE=zc706-zynq7
I am NOT using mickledore, but the latest honister. In order to build I commented out the line in the conf file requiring mickledore. This may be related...

When building python3-tensorflow-lite I get the following error,
`/usr/src/debug/python3-tensorflow-lite/2.11.0-r0/build/cpuinfo/src/init.c:58: undefined reference to 'cpuinfo_arm_linux_init'`

If I comment out this line the build progresses and finishes. 

Seeing similar issues in other projects : https://github.com/microsoft/onnxruntime/issues/8425

Following those steps, explicitly telling it to compile for armv7 doesn't seem to make any difference.

Not too surprisingly trying to use it on the embedded board I receive the following error,

```
>>> import tflite_runtime.interpreter as tflite    

ImportError: /usr/lib/python3.9/site-packages/tflite_runtime/_pywrap_tensorflow_interpreter_wrapper.so: undefined symbol: cpuinfo_isa
```

Any ideas?

**Comment by NobuoTsukamoto** [comment]
I think you need to add HOST_ARCH to the python3-tensorflow-lite recipe.
https://github.com/NobuoTsukamoto/meta-tensorflow-lite/blob/honister/recipes-framework/tensorflow-lite/python3-tensorflow-lite_2.8.1.bb#L60-L63

Try adding `HOST_ARCH:zc706-zynq7 = "armv7"`.

**Comment by gutelfuldead** [comment]
That looks like the right answer, I should have checked out the honister branch to begin with and that would have been more obvious. I got it running before I saw your response by modifying,

`/yocto/2022_2/build/tmp/work/cortexa9t2hf-neon-xilinx-linux-gnueabi/python3-tensorflow-lite/2.11.0-r0/build/cpuinfo/CMakeLists.txt`

and adding, `SET(CMAKE_SYSTEM_PROCESSOR "armv7")` which is essentially what you are saying to do, except not nearly as clean. 

Thanks for the help!
